docs/ProjectUsage.md

```diff
@@ -24,10 +24,10 @@ Approach 2: Simulating every scenario using unit tests are not feasible.
 ## What we propose
 #### The [VulnerableApp](https://github.com/SasanLabs/VulnerableApp):
 ```
-1. Simple SpringBoot Application which is very easy to run.
-2. Simple Annotation based and URL Path based vulnerable code execution
-3. Easy integration with any of the Web Application Vulnerability Finding Tools.
-4. Scenarios included are very common and exploitable using any Vulnerability finding tools.
+1. A simple web app built on top of the popular Spring framework
+2. Docker ready , can simply fire up the image and we are good to go
+2. Fully extensible , annotation and path based vulnerability execution

```

Comment:
Number type i think.

Reply:
woops :open_mouth:  my bad , 
Fixed this